static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.ts

```diff
@@ -41,101 +48,68 @@ const shouldRenderAdTestLabel = (): boolean =>
 		name: 'adtestInLabels',
 		shouldMemoize: true,
 	});
-
 // If `adtest` cookie is set, display its value in the ad label
-// Furthermore, provide a link to clear the cookie
-const createAdTestLabel = (): HTMLElement => {
-	const adTestLabel = document.createElement('span');
+const createAdTestLabel = (): string => {
+	let adTestLabel = '';
 
 	const shouldRender = shouldRenderAdTestLabel();
 	const val = getCookie({ name: 'adtest', shouldMemoize: true });
 
 	if (shouldRender && val) {
-		adTestLabel.innerHTML += ` [?adtest=${val}] `;
+		adTestLabel += ` [?adtest=${val}] `;
+		//the functionality to clear the adtest cookie will be reimplemented in a future ticket
```

Comment:
is this a leftover comment? This is implemented right?

Reply:
It is indeed, good spot!